spec.md

```diff
@@ -245,6 +243,34 @@ querying of the Service Broker's catalog, the Platform might not have an
 end-user with which to associate the request, therefore in those cases the
 originating identity header would not be included in those messages.
 
+## Service Broker Errors
+
+When a request to a Service Broker fails, the Service Broker MUST return an
+appropriate HTTP response code. If an appropriate response code is not defined
+within this specification, Service Broker authors SHOULD choose a code that
+accurately reflects the nature of the failure.
+
+For error responses, the following fields are defined. Service Brokers MAY
+include additional fields within the response. When adding new fields, Service
+Brokers SHOULD use a unique prefix for the field names to reduce the chances of
+conflict with future specification defined fields.
+
+| Response Field | Type | Description |
+| --- | --- | --- |
+| error | string | A single word in camel case that uniquely identifies the error condition. |
+| description | string | A user-facing error message explaining why the request failed. |
```

Comment:
I think we may need to add the standard "If present, MUST be a non-empty string." to these fields.